changelog/2021.04.29-rc2/changes/1514.md

```diff
@@ -0,0 +1 @@
+The `exporter.hits` metric has been removed.
```

Comment:
This is still to be fixed, currently it is still part of the output but always 0. I will open a bugfix PR later (or next week).
You can remove that file from this PR.

Reply:
I'll just fix it within this PR, it's not that hard to remove.

Reply:
Please check the last commit.

Reply:
Thanks!